main.tf

```diff
@@ -444,7 +479,11 @@ data "aws_iam_policy_document" "aggregated_policy" {
 }
 
 resource "aws_s3_bucket_policy" "default" {
-  count      = local.enabled && (var.allow_ssl_requests_only || var.allow_encrypted_uploads_only || length(var.s3_replication_source_roles) > 0 || length(var.privileged_principal_arns) > 0 || length(var.source_policy_documents) > 0) ? 1 : 0
+  count = local.enabled && var.cloudtrail_bucket_policy_enabled && (var.allow_ssl_requests_only ||
```

Comment:
this seems incorrect. this will only enable the bucket policy if the new input is true...

Reply:
Good catch -- too much line-break reformatting too early in the am..., will fix.